core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -471,4 +471,9 @@ private static Optional<TServerInstance> checkServer(ClientContext context, Stri
     }
     return server;
   }
+
+  @Override
+  public String toString() {
+    return getExtent() + " " + getDirName() + " " + getLocation();
```

Comment:
I would remove this, or move this if it's being used in the new code. TabletMetadata has so many fields and this is just printing a few. If someone modifies this method later, and it exposes data that we don't want exposed in the PrintMetadata utility, then that could be bad.